addme

```diff
@@ -0,0 +1,105 @@
+#/bin/bash
+
```

Comment:
Do all our systems have bash?  Have you tried /bin/sh and see if this still works?